sdks/python/apache_beam/runners/direct/clock.py

```diff
@@ -15,36 +15,37 @@
 # limitations under the License.
 #
 
-"""Clock implementations for real time processing and testing."""
+"""Clock implementations for real time processing and testing.
 
+For internal use only. No backwards compatibility guarantees.
+"""
 from __future__ import absolute_import
 
 import time
 
 
 class Clock(object):
-  """For internal use only; no backwards-compatibility guarantees."""
-
   def time(self):
     """Returns the number of milliseconds since epoch."""
-    return int(time.time() * 1000)
+    raise NotImplementedError()
+
+  def advance_time(self, advance_by):
+    """Advances the clock by a number of miliseconds."""
+    raise NotImplementedError()
 
 
-class MockClock(Clock):
-  """For internal use only; no backwards-compatibility guarantees.
+class RealClock(object):
+  def time(self):
+    return int(time.time() * 1000)
```

Comment:
<!--new_thread; commit:0f103d9eb9b51d9ac6f6b69380a720afd456ab3b; resolved:0-->
Can you please change this back to seconds?